### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by benjaminapetersen** [kubernetes-ui/container-terminal#8]
Quick suggestion since we are using the container terminal in the openshift web console as well now.  I see the "subject to change" disclaimer on the README.md file.  It is much easier to keep up with these changes if the releases lists `feature`, `bug`, `breaking change`, so wanted to toss that suggestion out there.

Thanks!

**Comment by stefwalter** [comment]
Good point. I'll remove the disclaimer, and switch to NVR style numbering to help with tracking breaking changes.

**Comment by benjaminapetersen** [comment]
Nice!  Is that similar to [semver](http://semver.org/)? I'll look it up, def will be helpful!

**Comment by stefwalter** [comment]
Er, I meant semver.
